source/includes/_courses.md

```diff
@@ -154,13 +154,11 @@ This endpoint retrieves all courses.
 
 Parameter | Description
 --------- | -----------
-filter[partner_id] | Scopes the courses returned to the partner specified.
-filter[role] | `student` will display all courses that the user is learning <br> `instructor` will display all courses the user is teaching
-filter[state] | `published` will display all courses that are in a published state <br> `unpublished` will display all courses that have not yet been published <br> `archived` will display all courses that have been archived
-page[number] | Courses are paginated, use this to choose which page you want
-page[size] | The default is `15` - How many courses are listed in each request/page
-tab | `published` will display all courses that are in a published state <br> `unpublished` will display all courses that have not yet been published <br> `archived` will display all courses that have been archived
+filter[partner_id] | Required. Scopes the courses returned to the partner specified.
+user_id | Specify a user id to get the courses that the user is enrolled in as a student. Defaults to the currently signed in user
+include_department_courses | If specified, courses in departments of `partner_id` that the user is enrolled in will also be returned
 sort | Orders the results based on the param. <br>Example: `name` - sort in alphabetical order
+page[number] | Courses are paginated, use this to choose which page you want
```

Comment:
Oh snap does this mean we can paginate the CoursesPage now? And maybe try to refactor the learner_stats endpoint and its consumers to be paginated as well?

Reply:
Yep, I think the biggest bit of work will be updating our tables to work with a paginated endpoint

Reply:
Haven't looked too deep into it, but react-query has some pagination utilities built-in: https://react-query.tanstack.com/docs/guides/paginated-queries